include/zephyr/posix/sched.h

```diff
@@ -3,8 +3,8 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-#ifndef ZEPHYR_INCLUDE_POSIX_POSIX_SCHED_H_
-#define ZEPHYR_INCLUDE_POSIX_POSIX_SCHED_H_
+#ifndef ZEPHYR_INCLUDE_POSIX_SCHED_H_
+#define ZEPHYR_INCLUDE_POSIX_SCHED_H_
```

Comment:
While this is the right thing to do, this will be a breaking change for the Zephyr users, upgrading to 3.3 from an older Zephyr version, who relied on the availability of `posix_sched.h`.

There should be a dummy `posix_sched.h` that prints out a deprecation warning and includes `sched.h` (can be removed after a few releases).

Reply:
Fixed!